PATCH-P0-HACK-UPSTREAM: Silence compilator about no return

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: sql/sql_plugin.cc
===================================================================
--- sql/sql_plugin.cc.orig
+++ sql/sql_plugin.cc
@@ -3370,6 +3370,7 @@ bool sys_var_pluginvar::session_is_defau
     return getopt_ulonglong2double(option.def_value) == *(double*)value;
   default:
     DBUG_ASSERT(0);
+    return false;
   }
 }
 
